analyzers/its/sources/Net5/Net5/S1185.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Net5
```

Comment:
_Not related to this file:_ Let's add test case for S2259 to verify Symbolic Execution for `init` setters

```
    public int PropertySimple
    {
        get => 42;
        init
        {
            object o = null;
            field = o.ToString();   // Noncompliant
        }
    }
```